src/promptflow/promptflow/_sdk/entities/_run.py

```diff
@@ -589,6 +590,7 @@ def _to_rest_object(self):
             session_setup_mode=SessionSetupModeEnum.SYSTEM_WAIT,
             compute_name=compute_name,
             identity=identity_resource_id,
+            enable_multi_container=get_enable_multi_container_from_env(),
```

Comment:
is there any server side default for this?

Reply:
default value is false: [source](https://msdata.visualstudio.com/Vienna/_git/vienna/pullrequest/1284829?_a=files&path=/src/azureml-api/src/Designer/src/PromptFlow/Contracts/SubmitBulkRunRequest.cs)

Reply:
I mean if user not set the env key, server side has a logic determine whether to enable multi container feature. 
like <2.0.0 use singlie, > 2.0.0 use multi. 

If we always pass a setting here, does it mean server side no longer has that feature?

Reply:
Got your point. Given service-side already has a default value, we can skip this parameter in case they want to add some more logic.